$NetBSD: patch-af,v 1.1 2007/09/29 16:34:42 rillig Exp $

Variables that are only used in one file don't need to be extern.

--- wml_backend/p3_eperl/eperl_proto.h.orig	2002-10-18 23:48:38.000000000 +0200
+++ wml_backend/p3_eperl/eperl_proto.h	2007-09-29 18:31:45.000000000 +0200
@@ -53,7 +53,6 @@ extern void RememberINC(char *str);
 extern void mysighandler(int rc);
 extern void myinit(void);
 extern void myexit(int rc);
-extern struct option options[];
 extern int main(int argc, char **argv, char **env);
 
 /* eperl_parse.c */
